db/test/test_data.py

```diff
@@ -263,6 +264,11 @@ def test_add_get_model(self):
 
     def test_get_summary_data(self):
         pass
+    
+    def test_check_low_level_duplicates(self):
```

Comment:
you can use `@unittest.skip` to skip an incomplete test. It will report it as skipped in the output, so that we don't forget about finishing it